src/main/java/org/folio/service/OrganizationService.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.service;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.persist.CriterionBuilder;
+import org.folio.rest.persist.Criteria.Criterion;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.util.DbUtils;
+import org.folio.util.ResponseUtils;
+
+import javax.ws.rs.core.Response;
+
+public class OrganizationService {
+  private final Logger logger = LogManager.getLogger(OrganizationService.class);
+
+  private static final String ORGANIZATION_TABLE = "organizations";
+  private static final String BANKING_INFORMATION_TABLE = "banking_information";
+
+  private final PostgresClient pgClient;
+
+  public OrganizationService(PostgresClient pgClient) {
+    this.pgClient = pgClient;
+  }
+
+  public void deleteOrganizationById(String id, Handler<AsyncResult<Response>> asyncResultHandler) {
+    Criterion criterion = new CriterionBuilder().with("organizationId", id).build();
+    pgClient.withTrans(conn ->
+      conn.delete(BANKING_INFORMATION_TABLE, criterion)
+        .compose(res -> {
+          logger.info("Deleted {} records from table {}", res.rowCount(), BANKING_INFORMATION_TABLE);
```

Comment:
Would be good to start logs with method name - deleteOrganizationById::

Reply:
thanks, fixed